New package: GaetanSencie.Comptabiliteannuelle version 1.7.0.0

STENS66 opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/199192)

STENS66 committed:
New package: GaëtanSencie.Comptabilitéannuelle version 1.7.0.0

stephengillie commented:
Hi @STENS66,

> The head ref may contain hidden characters: "Ga\u00EBtanSencie.Comptabilit\u00E9annuelle-1.7.0.0-3d197afa-c642-4a07-9f93-a80f5ab3b374"

- Can this be fixed? 

Manual Validation ended with:
![image](https://github.com/user-attachments/assets/a13b7226-ee77-46c0-b0ad-831272a7866a)

- Does another installer switch exist, to accept the license - and does this mean that the manifest needs an `Agreements` section?

Trenly commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.installer.yaml:
```diff
@@ -0,0 +1,28 @@
+  Silent: /silent
+  SilentWithProgress: /passive
+  Interactive: "true"
+  InstallLocation: C:\Program Files\Comptabilité annuelle
```
A location should not be hardcoded. If the location is needed, then `InstallLocationRequired: true` should be specified and the install location should be the switch needed to set the install location

Trenly commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.installer.yaml:
```diff
@@ -0,0 +1,28 @@
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /passive
+  Interactive: "true"
```
```suggestion
```
The interactive switch shouldn't be needed

Trenly commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.locale.fr-BE.yaml:
```diff
@@ -0,0 +1,18 @@
+License: Property
+Copyright: © Gaëtan Sencie 2023 tous droits réservés
+ShortDescription: Comptabilité annuelle Setup
+Tags: []
```
```suggestion
```

Trenly commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.installer.yaml:
```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GaëtanSencie.Comptabilitéannuelle
```
While special characters are allowed, it may make it more difficult for users to find and install the package. I recommend considering omitting accents from the identifier such as `GaetanSencie.Comptabiliteannuelle`

STENS66 committed:
Update GaëtanSencie.Comptabilitéannuelle.installer.yaml

"I have modified the special characters in the PackageIdentifier 'ë' and 'é' as agreed.

I have also removed 'Interactive: true'."

STENS66 committed:
Update GaëtanSencie.Comptabilitéannuelle.locale.fr-BE.yaml

"I have removed the following instruction: Tags: [] as suggested and also removed the special characters in the PackageIdentifier.

STENS66 committed:
Update GaëtanSencie.Comptabilitéannuelle.yaml

I have removed the special characters in the PackageIdentifier as suggested.

stephengillie commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.installer.yaml:
```diff
@@ -0,0 +1,27 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GaetanSencie.Comptabiliteannuelle
```
Please update file name and path to match this change.

stephengillie commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.locale.fr-BE.yaml:
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: GaetanSencie.Comptabiliteannuelle
```
Please update file name and path to match this change.

stephengillie commented on manifests/g/GaëtanSencie/Comptabilitéannuelle/1.7.0.0/GaëtanSencie.Comptabilitéannuelle.yaml:
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.9.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+
+PackageIdentifier: GaetanSencie.Comptabiliteannuelle
```
Please update file name and path to match this change.

stephengillie reviewed (changes requested):
> Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 
'manifests\g\GaëtanSencie\Comptabilitéannuelle\1.7.0.0', expected 
'manifests\g\GaetanSencie\Comptabiliteannuelle\1.7.0.0'

STENS66 committed:
Update and rename GaëtanSencie.Comptabilitéannuelle.installer.yaml to GaetanSencie.Comptabiliteannuelle.installer.yaml

Path name correction with replacement of special characters 'ë' and 'é'.
